### PR TITLE
Overhaul vignettes and fix most important checks

### PR DESCRIPTION
**Pull request by nuest**

### CODE REVIEW

**Comment by nuest** [comment]
@EHJ-52n Feedback on the "development guidelines" in `DEV-README.md` welcome.

File: DEV-README.md [hunk 1]
```diff
@@ -1,85 +1,169 @@
 # sos4R developer documentation
 
-## Package structure
+This file contains information for developers of the [R](http://r-project.org/) package `sos4R`.
+Documentation for usres can be found in the package's vignette (see `browseVignettes("sos4R")`).
```
Comment (EHJ-52n) [inline DEV-README.md]:
```suggestion
Documentation for users can be found in the package's vignette (see `browseVignettes("sos4R")`).
```

File: DEV-README.md [hunk 2]
```diff
@@ -1,85 +1,169 @@
...
 
-sos4R follows the regular R extension package structure. General documentation about R package development can be found at the following two websites.
+<!-- part of this page previously published at https://wiki.52north.org/Geostatistics/Sos4R -->
```
Comment (EHJ-52n) [inline DEV-README.md]: Better provide a speaking link with some explanation like: 
```suggestion
[Ancient documentation](https://wiki.52north.org/Geostatistics/Sos4R) is available and should be considered if something is missing in this readme or before suggesting changes.
```

File: DEV-README.md [hunk 3]
```diff
@@ -1,85 +1,169 @@
...
-* R extension packages by Hadley: http://r-pkgs.had.co.nz/ (well written with tight coupling to devtools and RStudio)
-* R extension manual: http://cran.r-project.org/doc/manuals/R-exts.html (generic, extensive)
+## Contribute
 
-## /R
+Please get in touch with the [community contact](https://52north.org/research/rd-communities/geostatistics/)
+of the geostatistics community and read the [Get Involved page](https://52north.org/software/get-involved/) if you want to *become a contributor* -- contributions are welcome!
 
-The actual source files in the ``/R`` directory follow a naming schema:
+You can also dive right in and joint the chat room: [![Join the chat at https://gitter.im/52North/sos4R](https://badges.gitter.im/52North/sos4R.svg)](https://gitter.im/52North/sos4R)
```
Comment (EHJ-52n) [inline DEV-README.md]: Are you willing and is it possible for you to be always/often available in this gitter chatroom?
```suggestion
You can also dive right in and join the chat room: [![Join the chat at https://gitter.im/52North/sos4R](https://badges.gitter.im/52North/sos4R.svg)](https://gitter.im/52North/sos4R)
```

File: DEV-README.md [hunk 4]
```diff
@@ -93,4 +177,35 @@ SosParsingFunctions("TimeseriesKVP" = function() { ... })
...
+
+A new release shall be uploaded to CRAN after testing and under the following procedure:
+
+- Run the [tests](#tests)
+- Check the CI status
+  - https://travis-ci.org/52North/sos4R
+  - https://ci.appveyor.com/project/nuest/sos4r
+- Update version and date in `man/sos4R-package.Rd`
+- Update version in `DESCRIPTION`
+- Create a git tag with the version number
```
Comment (EHJ-52n) [inline DEV-README.md]:
```suggestion
- Create a git tag with the version number in the format `vx.y.z`, e.g `v3.5.56` (following the rules of [semantic versioning](https://semver.org/)
```

File: DEV-README.md [hunk 5]
```diff
@@ -93,4 +177,35 @@ SosParsingFunctions("TimeseriesKVP" = function() { ... })
...
+- Update NEWS file based on latest commits
+- Read and follow http://cran.r-project.org/web/packages/policies.html and http://r-pkgs.had.co.nz/release.html#release-check again
+- Run checks
+  - In the parent directory of the project: `R CMD build sos4R; R CMD check --as-cran sos4R_0.<version number>.tar.gz` should have no errors, warnings, or notes
```
Comment (EHJ-52n) [inline DEV-README.md]:
```suggestion
  - In the parent directory of the project: `R CMD build sos4R; R CMD check --as-cran sos4R_<version number>.tar.gz` should have no errors, warnings, or notes
```
Or is the leading `0.` an CRAN requirement?

File: DEV-README.md (hunk 2 above)
Comment (nuest) [reply DEV-README.md]: Please the the wiki page - it is empty, though of course users could check the wiki history. I think that is unlikely.

File: DEV-README.md (hunk 4 above)
Comment (nuest) [reply DEV-README.md]: Good idea to give an example, semantic version is linked to above already, slightly rephrased it.

**Comment by EHJ-52n** [comment]
@nuest you can merge this PR from my point of view.

**Comment by nuest** [comment]
Closing in favour of https://github.com/52North/sos4R/pull/110

**Pull request closed without merging**